do not let the tests try reading/locking usb hardware.  we also disable
this because the test will abort() if no usb support is found at all.

--- a/testsuite/sanei/sanei_usb_test.c
+++ b/testsuite/sanei/sanei_usb_test.c
@@ -832,6 +832,8 @@ main (int argc, char **argv)
   int detected, opened, i;
   SANE_Int dn[MAX_DEVICES];
 
+  return 77;
+
 #ifdef HAVE_LIBUSB
   printf ("\n%s built with old libusb\n\n", argv[0]);
 #endif
